ietf-optical-impairment-topology.yang

```diff
@@ -1264,17 +1287,122 @@ module ietf-optical-impairment-topology {
+      description
+        "This augment is only valid for 
+        Optical Impairment topology ";
+    }
+    description
+      "Augment the attributes for the 'from' LTP for the TE node 
+      connectivity matrix entry.";
+    list additional-ltp {
+      when "derived-from-or-self(../../../../../../"
+         + "nt:termination-point[nt:tp-id=current()/../tet:tp-ref]/"
+         + "optical-imp-topo:protection-type,"
+         + "'optical-imp-topo:otsi-protection')" {
+        description
+          "This list applies only to 'from' LTPs that support 
+          individual OTSi(G) protection.";
+      }
+      key "ltp-ref";
+      config false;
+      description
+        "The restricted list of the potential secondary LTPs that 
+        can be selected when the 'from' LTP of this connectivity 
+        matrix entry is selected as a working LTP.
+        
+        If this list is empty, all the other LTPs that can reach 
+        the 'to' LTP of this connectivity matrix entry can be 
+        selected as secondary LTPs.";
+      leaf ltp-ref {
+        type leafref {
+          path "../../../../../../../nt:termination-point/nt:tp-id";
+        }
+        description
+          "The reference to the potential secondary LTP that can be 
+          selected when the 'from' LTP of this connectivity matrix 
+          entry is selected as a working LTP";
+      }
+      leaf roadm-path-impairments {
+        type leafref {
+          path "../../../../../../../tet:te/tet:te-node-attributes/"
+             + "roadm-path-impairments/roadm-path-impairments-id";
+        }
+        config false;
+        description
+          "Pointer to ROADM optical impairments of the ROADM path 
+          between the secondary LTP and the 'to' LTP of this 
+          connectivity matrix entry.";
+      }    
+    }
+  } // augmentation connectivity-matrix from
+
+  augment "/nw:networks/nw:network/nw:node/tet:te/"
+        + "tet:te-node-attributes/tet:connectivity-matrices/"
+        + "tet:connectivity-matrix/tet:to" {
+    when "../../../../../../nw:network-types/tet:te-topology/"
+       + "optical-imp-topo:optical-impairment-topology" {
+      description
+        "This augment is only valid for 
+        Optical Impairment topology ";
+    }
+    description
+      "Augment the attributes for the 'to' LTP for the TE node 
+      connectivity matrix entry.";
+    list additional-ltp {
+      when "derived-from-or-self(../../../../../../"
+         + "nt:termination-point[nt:tp-id=current()/../tet:tp-ref]/"
+         + "optical-imp-topo:protection-type,"
+         + "'optical-imp-topo:otsi-protection')" {
+        description
+          "This list applies only to 'to' LTPs that support 
+          individual OTSi(G) protection.";
+      }
+      key "ltp-ref";
+      config false;
+      description
+        "The restricted list of the potential secondary LTPs that 
+        can be selected when the 'from' LTP of this connectivity 
```

Comment:
shouldn't this be 'to' instead of 'from' ?

Reply:
Fixed in https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-optical-impairment-topology-yang/pull/116/commits/91193f0458ddb0b0bf3c4ec40e73356df546d39e